release_tester/test_suites/misc/binary_test_suite.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+"""this test suite is intended to perform all checks of the binary files, e.g. stripping, signatures etc."""
+import semver
+
+from arangodb.installers import create_config_installer_set
+from reporting.reporting_utils import step
+from test_suites_core.base_test_suite import parameters, testcase, run_after_each_testcase, run_before_suite
+from test_suites_core.cli_test_suite import CliStartedTestSuite, CliTestSuiteParameters
+
+
+class BinaryComplianceTestSuite(CliStartedTestSuite):
+    def __init__(self, params: CliTestSuiteParameters):
+        min_version = semver.VersionInfo.parse("3.12.0-nightly")
+        super().__init__(params)
+        if self.new_version is not None and semver.VersionInfo.parse(self.new_version) < min_version:
+            self.__class__.is_disabled = True
+            # pylint: disable=no-member
+            self.__class__.disable_reasons.append("Test suite is only applicable to versions 3.12 and newer.")
+        self.installer_set = create_config_installer_set(
+            versions=[self.new_version],
+            base_config=self.base_cfg,
+            deployment_mode="all",
+            run_properties=self.run_props,
+            use_auto_certs=False,
+        )
+        self.installer = self.installer_set[0][1]
+        ent = "Enterprise" if self.run_props.enterprise else "Community"
+        self.suite_name = f"Binary compliance test suite: ArangoDB v. {str(self.new_version)} ({ent}) ({self.installer.installer_type})"
+        self.binary_file_names = [
```

Comment:
hm, the installer should have the list of installed binaries and their locations. wouldn't it be better to use that? 
Also most probably the go-binaries don't need to be tested, hence a flag should be added to the binary registrations